container-core/src/main/java/com/yahoo/container/handler/threadpool/ExecutorServiceWrapper.java

```diff
@@ -28,10 +28,12 @@
     private final Thread metricReporter;
     private final AtomicBoolean closed = new AtomicBoolean(false);
 
-    ExecutorServiceWrapper(
```

Comment:
I don't like this style change either. The new style requires parameters to be reindented if `ExecutorServiceWrapper` is renamed or access modifier is introduced. It's annoying when a developer relies on automatic refactoring features in an IDE. It's also prone to create lines exceeding recommended max line length if either class name or parameter definitions are too long.

Reply:
I know you don't like it, and I see it as a taste thing where there's no right answer (the IDE can indent like this for you so it doesn't cost any time either way). But I'm an author on this file and you are not :-)

Reply:
Does it matter who is formally listed as author of the file? Many source files are modified by several developers that are not listed as author.

Reply:
I usually check authors if I want to change to a style I know people don't agree to. It just seems polite to me.

Btw, on author: I usually set myself as author if I make substantial changes to something.

None of these things are precise enough to be "rules" of course ...

Reply:
I don't agree in changing code style if there is no intention to do functional changes to the same code - we do not have any agreed style convention or automatic linting for our Java code. It clutters the git history and creates potential merge conflicts. Is it constructive if several developers reformat each others code all the time because they don't the other style?

Reply:
No but I don't think that is an actual problem we have. It's more that we have lots of code that looks a bit old and crufty with lots of yellow when viewed in IntelliJ.

(Not that that's a big problem either - here I was killing time while in a meeting.)